contracts/fast/lib/AFastFacet.sol

```diff
@@ -100,4 +113,37 @@ abstract contract AFastFacet is IFastEvents {
     );
     _;
   }
+
+  /** @dev Ensures that the given address is a member of the current FAST or the Zero Address.
```

Comment:
Isn't this a bit "too specific" to be in all facets? My intent with the top-level abstract facet was to factorize reusable stuff **across facets**.

Reply:
Yep, I was torn on this one. On the one hand, it's specific... and yep, probably only of use in the `FastTokenFacet`. But on the other, I was thinking about keeping modifiers in the same place etc from a maintenance point of view - probably overkill though.

Reply:
I've moved it back 🙂